src/demos/calcite-shell.html

```diff
@@ -566,31 +559,36 @@ <h2 class="heading">My App</h2>
       <p style="text-align: center;">Box Shadow added to show app edges.</p>
       <script>
         var actionPad = document.getElementById("action-pad");
+        var calcitePositionActionPad = document.getElementById("calcite-position-action-pad");
         var actionPadButton = document.getElementById("action-pad-button");
         actionPadButton.addEventListener("click", function() {
           actionPad.hasAttribute("hidden") ? actionPad.removeAttribute("hidden") : actionPad.setAttribute("hidden", "");
-
-          actionPad.positionElement = actionPadButton;
+          calcitePositionActionPad.positionElement = actionPadButton;
+          calcitePositionActionPad.reposition();
```

Comment:
Doesn't this kick in when `positionElement` is set?

Reply:
Yep. will remove